model-optimizer/mo/ops/broadcast.py

```diff
@@ -69,12 +69,18 @@ def infer(node: Node):
                 node.out_port(0).data.set_value(uni_directional_broadcasting(input_value, target_shape))
             elif node.mode == 'bidirectional':
                 node.out_port(0).data.set_value(bi_directional_broadcasting(input_value, target_shape))
+            elif node.mode == 'explicit' and node.in_port(2).data.get_value() is not None:
+                axes_mapping = node.in_port(2).data.get_value()
+                node.out_port(0).data.set_value(explicit_broadcasting(input_value, target_shape, axes_mapping))
             else:
                 raise Error('The node "{}" has unsupported mode "{}"'.format(node_name, node.mode))
         else:
             if node.mode == 'numpy':
                 node.out_port(0).data.set_shape(uni_directional_shape_broadcasting(input_shape, target_shape))
             elif node.mode == 'bidirectional':
                 node.out_port(0).data.set_shape(bi_directional_shape_broadcasting(input_shape, target_shape))
+            elif node.mode == 'explicit' and node.in_port(2).data.get_value() is not None:
+                axes_mapping = node.in_port(2).data.get_value()
+                node.out_port(0).data.set_shape(explicit_shape_broadcasting(input_shape, target_shape, axes_mapping))
             else:
                 raise Error('The node "{}" has unsupported mode "{}"'.format(node_name, node.mode))
```

Comment:
I see that PermuteInputs call is missing for the axes_mapping.

Reply:
Agree. Added PermuteInputs.